clients/rust/Cargo.toml

```diff
@@ -10,7 +10,7 @@ license-file = "../../LICENSE"
 [features]
 anchor = ["dep:anchor-lang"]
 test-sbf = []
-serde = ["dep:serde", "dep:serde_with"]
+serde = ["dep:serde", "dep:serde_with", "kaigan/serde"]
```

Comment:
@lorisleiva this optional feature-flag for `kaigan` is only necessary when the user adds `kaigan` types to the repository. This is a manual step, right? Would they be configuring Codama to use `kaigan` as well?

Asking because it doesn't seem like there's a good place to work this into create-solana-program.

Reply:
Yes this is a manual step that you only need to do if you end up using more complex types that are defined in Kaigan.